README.md

```diff
@@ -15,6 +15,51 @@ Machine learning potentials can be installed directly from [scripts/](scripts).
 - Units are: distance (Å), energy (eV), force (eV Å-1), time (fs)
 - Training using molecular mechanics (MM) is not supported as we've not found it to be efficient
 
+
+## Using with OpenMM
+
+The OpenMM backend only works with MACE at the moment
+
+First install requirements from conda-forge
+
+```
+mamba install -c conda-forge autode=1.1* python=3.9 openmm-torch=1.1 pytorch=2.0
```

Comment:
We should just add `openmm-torch` to `environment_mace.yml` to simplify these instructions.